src/runtime/cmd/kata-runtime/kata-check_ppc64le_test.go

```diff
@@ -47,8 +47,8 @@ func TestCCCheckCLIFunction(t *testing.T) {
 	}
 
 	moduleData := []testModuleData{
-		{filepath.Join(sysModuleDir, "kvm"), false, "Y"},
-		{filepath.Join(sysModuleDir, "kvm_hv"), false, "Y"},
+		{filepath.Join(sysModuleDir, "kvm"), "", true},
+		{filepath.Join(sysModuleDir, "kvm_hv"), "", true},
```

Comment:
Shouldn't that be `false`?

Reply:
@Jakob-Naucke , the boolean value is for representing if it is a directory and it is one right? `/sys/module/kvm`